add missing variables as expression

adimiz1 opened the pull request:
### Brief Summary of Changes
Add `angle` variable as expression

#### What does this PR address?
- [ ] GitHub issue (Add reference - #XX)
- [ ] Refactoring
- [x] New feature
- [ ] Bug fix
- [x] Adds more tests

#### Are tests included?
- [x] Yes
- [ ] No

#### Reviewer, please note:

#### Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I ran the full test suite before pushing the changes and all the tests pass.

patrick-tolosa commented:
@adimiz1 I'm not sure I understand this PR.

The title and the code changes don't quite align.
Can you please update the title and description to explain what was done, and what approach was taken to make the changes?

adimiz1 commented:
The following variables were missing as variables: 
`angle`, `zoom`, `opacity`, `radius`, `dpr`, `dps`, `quality`
Added them to the `CLDExpression` class so they can be used in the following syntax (which is part of the iOS SDK):
`let expression = CLDExpression.angle().multiple(by: 10)`
And so we can perform operations (add, substract, multiply etc...) on them (tests included)
Added tests to make sure they dont get normalized and can be parsed as expressions
